Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class GameManager : MonoBehaviour
+{
+    private bool _keyCollected;
+    private GameObject _keyObject;
+    private GameObject _keyUI;
+
+    void Start() {
+        _keyCollected = false;
+        _keyObject = GameObject.Find("Glitch(Key)");
+        _keyUI = GameObject.Find("UIKey");
+        
+    }
+
+    void Update() {
+        if (!_keyObject.activeSelf)
+            _keyCollected = true;
+
+        if(_keyCollected) {
```

Comment:
Space between `if` operator and opening parenthesis

Reply:
Fixed spacing in if statement